PS8lite.Rmd

```diff
@@ -188,6 +189,22 @@ Fit one Random Forest model `model_rf` using any approach you want.
 
 ```{r}
 
+## Fitting Random Forest model directly
+# tried different values for mtry and ntree
+# the values I chose gave me a better error estimate than the one in sample code
+
+# Fit random forest:
+model_rf <- randomForest(
+  # Model formula
+  form = model_formula,
+  # Training data
+  data = training, 
+  # At each node of tree, number of features/predictor variables to randomly
+  # choose from for splitting:
+  mtry = 2,
+  # Number of bagged (bootstrap aggregated) trees in your forest:
+  ntree = 150 
+)
```

Comment:
Hi, everything looks good. Here is an example comment that I would leave as part of a “code review” if this were real life. Then you would make the requested changes to this PR and then I would merge your changes into my master copy of the repo. I won't be doing that last step in this case.